pysheaf.py

```diff
@@ -10,19 +10,20 @@
 import matplotlib.pyplot as plt
 import networkx as nx
 import scipy.optimize
+import copy
 
 ## Data structures
 class Coface: 
     """A coface relation"""
-    def __init__(self,index,orientation):
+    def __init__(self,index=None,orientation=None):
```

Comment:
Here's one thing that's important.  In CellComplex, cofaces are "just" a relation between cells.  However for Sheaf, they contain additional data.  So check out my latest commit that allows you to avoid creating Coface (or SheafCoface) objects outside of a CellComplex (or Sheaf).